Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -9821,6 +9821,7 @@
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				BC18C4140E16F5CD00B34460 /* JavaScriptCore.h in Headers */,
```

Comment:
This seems unrelated?

Reply:
Fallout from running sort-Xcode-project-file, which I don't need to do since I'm not changing adding/removing files in the project.

Reply:
Ah, actually, it's getting run automatically (via the `pre-commit` hook), because I'm committing changes to the pbxproj. Probably best to just accept the change it's making.

The reason it's doing the reordering in the first place is because of 8cd33528ad547c1bec218e3b05d01e1d354c1b2e, specifically:
> Work around a header-ordering bug in tapi (rdar://104432605) by sorting all umbrella headers to the top of their targets' Headers phase.